x-pack/legacy/plugins/infra/public/components/logging/log_analysis_setup/initial_configuration_step/initial_configuration_step.tsx

```diff
@@ -39,20 +41,25 @@ export const InitialConfigurationStep: React.FunctionComponent<InitialConfigurat
   endTime,
   isValidating,
   validatedIndices,
+  setupStatus,
   setValidatedIndices,
   validationErrors = [],
 }: InitialConfigurationStepProps) => {
+  const disabled = useMemo(() => !editableFormStatus.includes(setupStatus), [setupStatus]);
```

Comment:
What's changing this setupStatus value and where?

Reply:
It's being passed in as a property to the component. It originates from the job management hook that uses a combination of the ML API responses and the setup flow state to determine the overall setup state.

Reply:
More precisely, this is the file with the reducer that derives the `setupState`:  https://github.com/weltenwort/kibana/blob/d7bc217827d19aced14fe3f47b83a85567a41b96/x-pack/legacy/plugins/infra/public/containers/logs/log_analysis/log_analysis_module_status.tsx

Reply:
Ah ok so this value was already being set, gotcha. Looks good then!